plugin.video.embycon/resources/lib/utils.py

```diff
@@ -118,7 +118,7 @@ def getPlayUrl(self, id, media_source, force_transcode, play_session_id):
                        "&MediaSourceId=%s")
             playurl = playurl % (server, id, play_session_id, media_source_id)
             user_token = downloadUtils.authenticate()
-            playurl = playurl + "&api_key=" + user_token
+            playurl = playurl + "&api_key=" + user_token + "|verifypeer=false"
```

Comment:
same

Reply:
again same, can probably be changed.
however again it wont get to this stage as the API requests are still checks and no play url can be built without the API call first.

Reply:
use the verify_cert setting to append the verifypeer=false is needed